src/wkwebkit.cs

```diff
@@ -493,16 +493,19 @@ interface WKUIDelegate {
 		void DidClose (WKWebView webView);
 
 		[iOS (10,0)][NoMac]
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'TBD' instead.")]
```

Comment:
I wondered a while where `TBD` was... so maybe phrase is without an abbreviation? "Deprecated API, the replacement API will be implemented later" or something along those lines?

Reply:
It's not meant to be released (final) like this :) but it follows the final format
Added to https://github.com/xamarin/xamarin-macios/issues/6212